core/templates/services/utils.service.ts

```diff
@@ -114,7 +114,6 @@ export class UtilsService {
   isError(value: Object): boolean {
     switch (Object.prototype.toString.call(value)) {
       case '[object Error]': return true;
-      case '[object Exception]': return true;
```

Comment:
There is no Exception class in Typescript. Hence we cannot get an object of the exception class.

Please Note:
`throw new Error('Error')` results in an object of Error class